jlm/llvm/ir/operators/load.cpp

```diff
@@ -11,31 +11,32 @@
 namespace jlm::llvm
 {
 
-LoadOperation::~LoadOperation() noexcept = default;
+LoadNonVolatileOperation::~LoadNonVolatileOperation() noexcept = default;
 
 bool
-LoadOperation::operator==(const operation & other) const noexcept
+LoadNonVolatileOperation::operator==(const operation & other) const noexcept
 {
-  auto operation = dynamic_cast<const LoadOperation *>(&other);
+  auto operation = dynamic_cast<const LoadNonVolatileOperation *>(&other);
   return operation && operation->narguments() == narguments()
       && operation->GetLoadedType() == GetLoadedType()
       && operation->GetAlignment() == GetAlignment();
 }
 
 std::string
-LoadOperation::debug_string() const
+LoadNonVolatileOperation::debug_string() const
 {
   return "LOAD";
```

Comment:
I guess the debug string can remain as "LOAD", but would you want to change the file name of this file as a part of this PR?

Reply:
@haved I changed the debug string to "Load". Moreover, I changed the file names to `LoadNonVolatile.[ch]pp` as well as the test file name to `LoadNonVolatileTests.cpp`. Please have a look again.